channels/in.m3u

```diff
@@ -90,7 +98,7 @@ https://dai.google.com/linear/hls/event/IdHTuehZQPClis-gJaZkFQ/master.m3u8
 #EXTINF:-1 tvg-id="Channel.in" tvg-name="Channel" tvg-country="IN" tvg-language="" tvg-logo="https://static.epg.best/in/ChannelV.in.png" group-title="",Channel [V]
 https://livecdn.fptplay.net/foxlive/channelvhd_hls.smil/chunklist_b2500000.m3u8
```

Comment:
I don't know how to suggest changes, use user-agent from yesterday to revive it

Reply:
Going to add that when I can, and just look at this little icon over here.

![image](https://user-images.githubusercontent.com/30985701/125540347-c1d8477e-ea6a-4f48-b7b1-e457feb102c2.png)